Library/.rubocop.yml

```diff
@@ -8,7 +8,15 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+
 Homebrew/CorrectBottleBlock:
+  Include:
+    - '**/Taps/homebrew/**/*.rb'
```

Comment:
Well, I was trying to keep the cop code and config code separate in their respective files. But turns out, keeping the `Include` logic in `FormulaCop` is more elegant. Thanks @reitermarkus and @MikeMcQuaid, now Personal Taps can be passed as arguments. 
I added the logic in Formula Cop and pushed the new code. I also used rebase to add another test in bottle_block_cop_spec in a past commit to pass coverage test.